Update cs-rin-ru-enhanced-mod.user.js

Altansar69 opened the pull request:
#94

Altansar69 committed:
Update cs-rin-ru-enhanced-mod.user.js

#94

Reddiepoint reviewed (changes requested):
SteamDB part looks good, but the mentioning change is not working with the quick reply panel. You also need to replace the postBody at line 557 with "let postBody = \`@[b][url=${FORUM_BASE_URL}memberlist.php?mode=viewprofile&u=${authorID}]${decodeURI(author)}[/url][/b] \`;"

Reddiepoint committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into beta

# Conflicts:
#	cs-rin-ru-enhanced-mod.user.js

Reddiepoint committed:
Enhance post preview function: change from `postlink-local` to `postlink` so the code applies to more links in the forum (mainly links in signatures and links in "X wrote:" quotes - useful to see full post if contents are snipped).
Added code for previewing topics which are linked (limited to first post only).

Bump version to 1.2.7

Reddiepoint committed:
Add back Altansar's changes (accidentally got lost in the merge) - change SteamDB Link label to be consistent with BBCode and change mentioning to bold the username
